lib/sql-string.js

```diff
@@ -35,6 +35,11 @@ function escape(val, timeZone, dialect, format) {
     break;
   }
 
+  // moment
+  if (val.toDate) {
```

Comment:
If we can't assume `toDate` returns a `Date`, by the same logic how can we assume the object is a moment when `isMoment` returns true?